### PR TITLE
IDVA5-448-What is your name test branch

### PR DESCRIPTION
**Pull request by atedros-ch**
- gov.uk links and scripts added for CSS
- new design implement
- validation separate from common
- new test case added and updated

### CODE REVIEW

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

**Review by abhiahirv1 — DISMISSED** [review]
LGTM

File: src/test/src/controllers/soleTraderWhatIsTheBusinessName.test.ts [hunk 1]
```diff
@@ -18,7 +18,8 @@ describe("GET" + SOLE_TRADER_WHAT_IS_THE_BUSINESS_NAME, () => {
 describe("POST" + SOLE_TRADER_WHAT_IS_THE_BUSINESS_NAME, () => {
     it("should redirect with status 302 on successful form submission", async () => {
         const formData = {
-            whatIsTheBusinessName: "Company"
+            whatIsTheBusinessNameInput: "Cpmpany",
```
Comment (sampankumar) [inline src/test/src/controllers/soleTraderWhatIsTheBusinessName.test.ts:21]: Spelling mistake probably?

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

**Review by abhiahirv1 — APPROVED** [review]

**Review by sampankumar — APPROVED** [review]

**Pull request merged**